test_regress/t/t_cdc_async_bad.pl

```diff
@@ -16,7 +16,7 @@
     make_top_shell => 0,
     make_main => 0,
     fails => 1,
-    expect_filename => $Self->{golden_filename},
+    expect_filename => $Self->{dynamic} ? $Self->{golden_filename} =~ s/\.out$/_dyn.out/r : $Self->{golden_filename},
```

Comment:
The `--cdc` option is deprecated, so maybe we can just not run this test with delays, though I'm not sure why this changes output?

Reply:
The output is changed due to the output directory being named after the scenario. Now I see that this is still not the correct check. Though I guess we want to move away from scenarios, so consider this no longer relevant.